indico/modules/events/registration/templates/display/event_header.html

```diff
@@ -78,12 +78,84 @@
 
 {% if published_registrations %}
     <div class="event-details-row">
-        <div class="event-details-label">{% trans %}Participants{% endtrans %}</div>
+        <h2 class="event-details-label">{% trans %}Participants{% endtrans %}</h2>
         <div class="event-details-content">
-            <div class="participant-list-wrapper">
-                <ul class="participant-list">
+            <ind-collapsible  lines="3" collapsed class="participant-list-wrapper">
+                <ul class="participant-list" data-collapsible>
                     {%- for participant in published_registrations|sort(attribute='display_full_name') -%}
-                        <li class="ui image label meeting-participant {% if participant.checked_in and participant.registration_form.publish_checkin_enabled %} checked-in {% endif %}">
+                        <li class="meeting-participant {% if participant.checked_in and participant.registration_form.publish_checkin_enabled %} checked-in {% endif %}">
```

Comment:
Don't forget to remove that testing stuff here ;)

Reply:
BTW an easier way to test with repeated participants is this:

```
{%- for participant in published_registrations|sort(attribute='display_full_name') * 4 -%}
```

Reply:
Thanks, I totally forgot you could do that in Python. 😂